tests/test_prompt_tokenizers.py

```diff
@@ -53,6 +56,45 @@ def test_sharegpt_integration(self):
             self.assertEqual(len(example[fields]), len(tokenized_conversation[fields]))
             self.assertEqual(example[fields], tokenized_conversation[fields])
 
+    def test_completion(self):
```

Comment:
I think this function name could be renamed as the below does not use the `completion` prompter?

Reply:
thanks, I had it in a completion class, then renamed and omitted things from the PR. will adjust